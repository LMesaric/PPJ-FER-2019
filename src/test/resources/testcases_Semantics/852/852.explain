semantic: conflicting declarations of function p
note: 'p ( int a , int b )' on line 4
