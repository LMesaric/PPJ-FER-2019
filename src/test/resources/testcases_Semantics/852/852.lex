KR_INT 2 int
IDN 2 main
L_ZAGRADA 2 (
KR_VOID 2 void
D_ZAGRADA 2 )
L_VIT_ZAGRADA 2 {
KR_VOID 3 void
IDN 3 p
L_ZAGRADA 3 (
KR_INT 3 int
IDN 3 a
D_ZAGRADA 3 )
TOCKAZAREZ 3 ;
KR_VOID 4 void
IDN 4 p
L_ZAGRADA 4 (
KR_INT 4 int
IDN 4 a
ZAREZ 4 ,
KR_INT 4 int
IDN 4 b
D_ZAGRADA 4 )
TOCKAZAREZ 4 ;
KR_INT 5 int
IDN 5 a
OP_PRIDRUZI 5 =
BROJ 5 12
TOCKAZAREZ 5 ;
KR_INT 6 int
IDN 6 polje
L_UGL_ZAGRADA 6 [
BROJ 6 4
D_UGL_ZAGRADA 6 ]
OP_PRIDRUZI 6 =
L_VIT_ZAGRADA 6 {
BROJ 6 1
ZAREZ 6 ,
ZNAK 6 'c'
ZAREZ 6 ,
BROJ 6 3
D_VIT_ZAGRADA 6 }
TOCKAZAREZ 6 ;
IDN 7 a
OP_PRIDRUZI 7 =
BROJ 7 3
PLUS 7 +
BROJ 7 3
TOCKAZAREZ 7 ;
D_VIT_ZAGRADA 9 }
KR_VOID 11 void
IDN 11 p
L_ZAGRADA 11 (
KR_INT 11 int
IDN 11 a
D_ZAGRADA 11 )
L_VIT_ZAGRADA 12 {
KR_RETURN 14 return
TOCKAZAREZ 14 ;
D_VIT_ZAGRADA 16 }
KR_VOID 17 void
IDN 17 p
L_ZAGRADA 17 (
KR_INT 17 int
IDN 17 a
ZAREZ 17 ,
KR_INT 17 int
IDN 17 b
D_ZAGRADA 17 )
L_VIT_ZAGRADA 17 {
KR_RETURN 18 return
TOCKAZAREZ 18 ;
D_VIT_ZAGRADA 19 }
