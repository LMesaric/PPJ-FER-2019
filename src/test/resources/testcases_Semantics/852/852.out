<izravni_deklarator> ::= IDN(4,p) L_ZAGRADA(4,() <lista_parametara> D_ZAGRADA(4,))
